src/api/initialDataTypes.js

```diff
@@ -181,8 +182,11 @@ export type InitialDataRealm = $ReadOnly<{|
   realm_bot_creation_policy: number,
   realm_bot_domain: string,
 
-  // TODO(server-3.0): Added in feat. 11
-  realm_community_topic_editing_limit_seconds?: number,
+  // TODO(server-8.0): Added in feat. 225
+  realm_can_access_all_users_group?: boolean,
```

Comment:
```suggestion
  // TODO(server-8.0): Added in feat. 225
  realm_can_access_all_users_group_id?: boolean,
```

Right?

… Oh huh, the changelog and the /api/register-queue page disagree. The latter says `…_group`, so I guess that's what it actually is.

Reply:
Huh, interesting, yeah.